### DISCUSSION TITLE
The app launches but then closes and relaunches as the normal version direct from steam.

### DISCUSSION THREAD

**Discussion by woodenlalaloo** [SteamRE/DepotDownloader#325] [Q&A]
Ok, this is the last one I promise. I downloaded the app, and it is a mac app so that's all good but when I launch it closes and then launches as the version I have installed directly from steam. It doesn't even show a window, it just closes then reopens as the newer version from steam.

**Comment by Skippeh** [comment]
Try creating steam_appid.txt in the root of the game folder and write the steam app id inside. Without it steamworks api won't be able to initialize if it's not launched through steam. Alternatively i think an environment variable ```SteamAppId``` could work too but i could be wrong. The text file method is easier either way.

Some games will check for this file and remove it as a form of DRM (since you could technically spoof it to another game you own) so it won't work, but i've only encountered it in one game personally.
